main.go

```diff
@@ -8,7 +8,20 @@ import (
 	"github.com/deis/object-storage-cli/config"
 )
 
+var (
+	debugStrs = []string{"DEIS_DEBUG", "DEBUG"}
```

Comment:
Fixed in https://github.com/deis/object-storage-cli/pull/12/commits/0f5b28c699a160d8fb343e6c2d270987a78b9933